src/components/AddComponent/GitOptions.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { ExpandableSection, Stack, StackItem } from '@patternfly/react-core';
+import { useField } from 'formik';
+import { InputField } from '../../shared';
+
+export const GitOptions: React.FC = () => {
+  const [, { value: detectedComponents }] = useField('detectedComponents');
+  if (!detectedComponents) {
```

Comment:
I was a bit confused because in the marvel design, the section appears after the URL is entered. So I thought It had to be shown after validation.
Updated.

Reply:
It needs to be shown after validation of the URL in UI. Not validated by the detection query.